app/assets/stylesheets/conversation_show.scss

```diff
@@ -52,9 +82,18 @@ margin: 0 0 5px 0;
   font-size: 12px;
 }
 
-.message {
-  background-color: $ghost;
-  margin: 0;
+
+.chatboxmessagecontent p {
+  font-size: 12px;
+  margin: 0 0 0.2rem 0;
```

Comment:
Properties should be ordered font-size, -moz-hyphens, -webkit-hyphens, hyphens, margin, -ms-word-break, word-break<br>Shorthand form for property `margin` should be written more concisely as `0 0 0.2rem` instead of `0 0 0.2rem 0`